doc/man/scons.xml

```diff
@@ -6201,9 +6252,10 @@ echo Last build occurred  . &gt; $TARGET
 
 <refsect2 id='python_code_substitution'><title>Python Code Substitution</title>
 
-<para>Any python code within
-<emphasis role="bold">${</emphasis>-<emphasis role="bold">}</emphasis>
-pairs gets evaluated by python 'eval', with the python globals set to
+<para>Any Python code within
+curly braces (and introduced by the variable prefix <literal>$</literal>)
```

Comment:
I'd leave the ${'s.
I've had people not know what curly braces are..

Reply:
Indeed.  It's remeniscent of the coffee holder on the front of most "desktop" computers, that silly technicians say you're supposed to put a CD into....

Reply:
Curly braces are spelled out just a bit earlier... line 5974 - the previous section. Is that good enough or do you want them added back here as well?

Reply:
I'd retain them here.